webapp/shop/views.py

```diff
@@ -56,10 +57,21 @@ def invoices_view(advantage_mapper: AdvantageMapper, **kwargs):
             filters={"marketplace": marketplace} if marketplace else None,
         )
 
+    if not rows_per_page:
+        rows_per_page = 10
+    if not page:
+        page = 1
+    payments_slice = slice(
+        (page - 1) * rows_per_page, (page * rows_per_page)
+    )  # because black and autopep8 don't agree on rules
+    payments_page = payments[payments_slice]
+
     return flask.render_template(
         "account/invoices/index.html",
-        invoices=payments,
+        invoices=payments_page,
```

Comment:
You can use the slice notation directly `invoices=payments[(page - 1) * per_page:page * per_page],`

Reply:
Using slices gives a linting error. It thinks that :(colon) is for a new block